azlinux-jupyter-scipy/start-notebook.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
i am wondering if maybe we should download all those files from the original github repo when we build the image, instead of copying them 🤔 Did you have to make any changes to files other than the dockerfile?

Reply:
Oh yes, the primary change is that all the original Docker images were built on an Ubuntu based machine, but we needed to switch them to be built on top of  Amazon Linux for FIPS compliance.

Reply:
I mean, could we wget `docker_healthcheck.py`, `fix-permissions`, `start-notebook.sh` etc straight from the original repo and build+run the container without modifying them?

Reply:
Oh yeah, that we can do. that seems like a nice idea. Let me make that change. 

Just a side note, I will use a permalink of the doc instead of the master branch's version, so that if there were any changes to the master branch that are incompatible with our dockerfile, things don't break.

Reply:
I removed my request for review while you look into this ^

Reply:
I've updated the architecture so that the support files get downloaded within the docker container during the building process instead of from local file-explorer.